doc/manual/src/glossary.md

```diff
@@ -74,36 +90,7 @@
   over the network, but cannot host builds. Examples of binary caches
   include S3 buckets and the [NixOS binary cache](https://cache.nixos.org).
 
-- [store path]{#gloss-store-path}
-
-  The location of a [store object] in the file system, i.e., an immediate child of the Nix store directory.
-
-  > **Example**
-  >
-  > `/nix/store/a040m110amc4h71lds2jmr8qrkj2jhxd-git-2.38.1`
-
-  See [Store Path](@docroot@/store/store-path.md) for details.
-
-  [store path]: #gloss-store-path
-
-- [file system object]{#gloss-file-system-object}
-
-  The Nix data model for representing simplified file system data.
-
-  See [File System Object](@docroot@/store/file-system-object.md) for details.
-
-  [file system object]: #gloss-file-system-object
-
-- [store object]{#gloss-store-object}
-
-  Part of the contents of a [store].
-
-  A store object consists of a [file system object], [references][reference] to other store objects, and other metadata.
-  It can be referred to by a [store path].
-
-  See [Store Object](@docroot@/store/index.md#store-object) for details.
-
-  [store object]: #gloss-store-object
```

Comment:
I can be convinced, but I felt like in the limit that means that every intra-manual link will indirect through the glossary, which feels unwieldy. When I am reading wikipedia I am happy for the link to take me directly to a page, and I figured I would be here too. It can also remind us that each page should start with a punchy sentence that stands alone.

(If we could automatically populate a glossary with said punchy first sentences, I might like it better.)

Reply:
I agree that we should just link to the page basically always.

What if we keep one-sentence descriptions in the glossary? That way it remains useful as a "cheat sheet" or overview of concepts or index. They'll be low maintenance, but valuable.

> automatically populate

Would have been easy with docbook, adding an attribute and processing that. With markdown I'm not so sure.
Maybe just keep it low tech and put a comment like this?
`<!-- glossary "store path" links here -->`
I'm also ok with doing nothing at all for this purpose, because I don't think we'll accidentally destroy introductions, and it'd be an easy fix.

Reply:
> I agree that we should just link to the page basically always.

> What if we keep one-sentence descriptions in the glossary? 

OK fair compromise, I am doing that